download_sra.nf

```diff
@@ -428,7 +428,7 @@ fi
 process extractSRA {
     container "quay.io/fhcrc-microbiome/get_sra:v0.4"
     label "io_limited"
-    errorStrategy 'retry'
+    errorStrategy 'ignore'
```

Comment:
Should this be `ignore` or `finish`? @jgolob

Reply:
Good catch. I fixed this and others.